server/datastore/mysql/operating_system_vulnerabilities_test.go

```diff
@@ -518,3 +527,105 @@ func testListKernelsByOS(t *testing.T, ds *Datastore) {
 		})
 	}
 }
+
+func testKernelVulnsHostCount(t *testing.T, ds *Datastore) {
```

Comment:
This is the "one kernel -> many OSs" test case. Also threw in a couple teams to make things interesting, found some issues that way 🎉